admin-ui/plugins/auth-server/redux/api/ScriptApi.js

```diff
@@ -0,0 +1,66 @@
+import { handleResponse } from 'Utils/ApiUtils'
+
+export default class ScriptApi {
+  constructor(api) {
+    this.api = api
+  }
+
+  // Get all scripts
+  getAllScripts = (options = {}) => {
+    return new Promise((resolve, reject) => {
+      this.api.getConfigScripts(options, (error, data) => {
+        handleResponse(error, reject, resolve, data)
+      })
+    })
```

Comment:
Dropping the full `response` object in a callback chain is bad because it violates the principle of robust error handling by failing to propagate a function argument to the next handler, which discards valuable context like HTTP status codes and headers, thereby limiting the downstream handler's ability to react to different failure scenarios.
This also applies to: 25-28, 34-37, 43-46, 52-55, 61-64